clicommand/env_dump.go

```diff
@@ -9,29 +9,31 @@ import (
 	"github.com/urfave/cli"
 )
 
-const envDescription = `Usage:
-  buildkite-agent env [options]
+var EnvDumpHelpDescription = `Usage:
```

Comment:
Why change const to var, and why change the capitalisation (making it exported)?

Reply:
Ah — I used the `clicommand/meta_data_*.go` as an example of a collection of subcommands, so followed the patterns therein rather than editing the old `env.go`. I guess the result was close enough that git decided it was an edit-diff rather than a delete-file / new-file.

Reply:
All the other collections-of-subcommands follow this pattern to, so I think I'll stick with it, and we can change them all at once if we prefer a different style.

Reply:
SGTM